special-use/special-use-POST-suds-mapping-post.json

```diff
@@ -100,10 +106,7 @@
             "note": "Stored in the middle layer.",
             "type": "file"
           },
-          "AUTOPOPULATED_BY_MIDDLEWARE_NOT_INTAKE_MODULE": { "basic-api-field": "PURPOSE", "note" : "This field should be a concatenation of activityDescription, locationDescription, startDateTime, endDateTime."},
-          "AUTOPOPULATED_BY_MIDDLEWARE_NOT_INTAKE_MODULE": { "basic-api-field": "CR_EXEMPT", "value" : "N"},
-          "AUTOPOPULATED_BY_MIDDLEWARE_NOT_INTAKE_MODULE": { "basic-api-field": "FORM_NAME", "value" : "FS-2700-3f"},
-         "required-by-basic-api": ["orgType","insuranceCertificate","goodStandingEvidence","operatingPlan"]
+          "AUTOPOPULATED_BY_MIDDLEWARE_NOT_INTAKE_MODULE": { "basic-api-field": "FORM_NAME", "basic-api-endpoint": "/contact/application", "value" : "FS-2700-3f", "required-by-basic-api": "Y"},
```

Comment:
The majority of the fields are applicant info and common application fields, which are the same between permits. I think I'd rather maintain a large file, rather than have to make applicant info field changes in two places.